src/utils/get-pack-nr1-url.js

```diff
@@ -24,9 +30,39 @@ const getPackNr1Url = (quickstartId, nerdletId, debug = false) => {
   const local = debug ? 'packages=local&' : '';
   const NR1_URL = debug ? NR1_BASE_URL_LOCAL : NR1_BASE_URL;
 
-  const url = new URL(`${NERDLET_PATH}?${local}pane=${hash}`, NR1_URL);
+  const packDetailsUrl = new URL(
+    `${NERDLET_PATH}?${local}pane=${hash}`,
+    NR1_URL
+  );
 
-  return url.href;
+  return packDetailsUrl.href;
 };
 
-module.exports = getPackNr1Url;
```

Comment:
This is the export statement we'd like to keep. It'll just need to be changed to a named export